src/main/java/gui/RootCenterArea.java

```diff
@@ -85,7 +85,8 @@ private void initMainTimeLinePane() {
         // main timeline panes
         mainTimelineScrollpane = new ScrollPane();
         mainTimeLineAnchorPane = new AnchorPane();
-        mainTimeLineGridPane = new TimelinesGridPane(numberOfTimelines, numberOfCounts, 1000, 1000);
```

Comment:
Removed most of them as they were unnecessary, and changed the rest to constants.